transfers/transfer.py

```diff
@@ -11,12 +11,15 @@
 import base64
 import logging
 import logging.config  # Has to be imported separately
+import models
```

Comment:
When replacing `import models` with `from . import models` getting the following error:
`ValueError: Attempted relative import in non-package`
I think it is related to http://stackoverflow.com/questions/11536764/attempted-relative-import-in-non-package-even-with-init-py
As we are running transfers.py as a standalone script, would it be better stay with `import models` ? Or does it break anything in python3 (I am testing with python 2.7)